Loop Status Extension/StatusViewController.swift

```diff
@@ -114,9 +114,9 @@ class StatusViewController: UIViewController, NCWidgetProviding {
             activeInsulinTitleLabel.textColor = .secondaryLabel
             activeInsulinAmountLabel.textColor = .label
         } else {
-            activeCarbsTitleLabel.textColor = .subtitleLabelColor
+            activeCarbsTitleLabel.textColor = .systemGray
```

Comment:
Does this work for dark mode as expected?

Reply:
So, this is for before iOS 13 (do we even need such checks any more since the deployment target is 13?). Should we consider dark mode for such colours?